added link to GMMK Website for initial flashing (#15806)
qmk/qmk_firmware: keyboards/gmmk/pro/iso/readme.md

* Hold the Reset switch mounted on the bottom side of the PCB while connecting the USB cable
* Hold the Escape key while connecting the USB cable (also erases persistent settings)
* Fn+Backspace will reset the board to bootloader mode if you have flashed the default QMK keymap

> * Check [GMMK website blog](https://www.pcgamingrace.com/blogs/news/step-by-step-guide-to-configuring-your-gmmk-pro-using-qmk) for further information on first time flashing your Pro-Keyboard 

Reviewer: ```suggestion
* Check [GMMK's blog page](https://www.pcgamingrace.com/blogs/news/step-by-step-guide-to-configuring-your-gmmk-pro-using-qmk) for more information on flashing your keyboard for the first time.
```
